zoom/plugin.spec.yaml

```diff
@@ -355,3 +355,36 @@ triggers:
         description: User Activity
         type: user_activity
         required: false
+tasks:
+  retrieve_user_activity_events:
+    title: Retrieve User Activity Events
+    description: Retrieve zoom user activity events
+    input:
+      activity_type:
+        title: Activity Type
+        description: Type of user activity to match event
+        type: string
+        required: true
+        enum:
+        - "Sign in"
+        - "Sign out"
+        - "All"
+        example: "All"
+    output:
+      user_activity_events:
+        title: User Activity Events
+        description: User Activity
+        type: '[]user_activity'
+        required: false
```

Comment:
Do we need `persist_state` here? I have added validation for `persist_state` in the icon-validator.

Reply:
`persist_state` will be optional.  If not there we won't do anything for the field

Reply:
Ok, I need to update the validator.